pkg/plugin/plugin.go

```diff
@@ -16,17 +16,75 @@ package plugin
 
 import (
 	"context"
+	"errors"
 	"fmt"
+	"strconv"
 
 	jvspb "github.com/abcxyz/jvs/apis/v0"
+	"github.com/abcxyz/pkg/githubapp"
+	"github.com/google/go-github/v55/github"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 )
 
-type GithubPlugin struct{}
+const (
+	// githubCategory is the justification category this plugin will be validating.
+	githubCategory               = "github"
+	respAnnotationKeyIssueOwner  = "github_issue_owner"
+	respAnnotationKeyIssueRepo   = "github_issue_repo"
+	respAnnotationKeyIssueNumber = "github_issue_number"
+)
 
-func (g *GithubPlugin) Validate(ctx context.Context, req *jvspb.ValidateJustificationRequest) (*jvspb.ValidateJustificationResponse, error) {
-	return nil, fmt.Errorf("Unimplemented")
+// issueMatcher is the mockable interface for the convenience of testing.
+type issueMatcher interface {
```

Comment:
This should live in validator class.

Reply:
Sorry for the churn, after a second look, since issueMatcher only used in plugin.go, please ignore this comment.